### PR TITLE
Fix softlinkstatic to actually create symlinks

### PR DESCRIPTION
**Pull request by jgmize**

### CODE REVIEW

**Comment by jgmize** [comment]
as pointed out by @stephendonner in IRC:
5:16 PM <•stephend> one is that loading.gif issue - http://www.webpagetest.org/result/150930_6B_1R8G/
5:16 PM <•stephend> (seems to be there, just with a dynamic asset-hashed filename
5:17 PM <•stephend> other is http://www.webpagetest.org/result/150930_6P_21WH/ - https://www.allizom.org/en-US/contact/spaces/
5:18 PM <•stephend> missing https://www.allizom.org/media/img/contact/moz-map-pin.png asset

There are still references to original, unhashed image filenames so we need to continue serving them from the /static dir.

File: docker/softlinkstatic.py [hunk 1]
```diff
@@ -7,10 +7,11 @@
...
-for path in static_files['paths']:
-    full_path = os.path.join('./static', path)
-
-    try:
-        os.unlink(full_path)
-    except OSError:
-        pass
+for orig, hashed in static_files['paths'].items():
+    if '?' in orig:
+        continue
```
Comment (pmclanahan) [inline docker/softlinkstatic.py]: would be nice to catch these too, but I get ya. Could just split on `?`, but my other idea for this doesn't see those query params, so we can fix that way.

Comment (jgmize) [inline docker/softlinkstatic.py]: Before I wrote it this way I validated that in every instance of an `orig` with a query param, there is already a corresponding `orig.split('?')[0]` in `static_files['paths']`, so I didn't see a point in processing those twice.

Comment (pmclanahan) [inline docker/softlinkstatic.py]: Oh cool. Of course you didn't, because there isn't one. Didn't realize that. Nice catch.

**Pull request merged**